experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -280,10 +284,29 @@ function rank(R::RootSystem)
 end
 
 function root_system_type(R::RootSystem)
-  @req has_root_system_type(R) "root system type not defined"
```

Comment:
What happens when the root system type is not known ?

Reply:
The next line errors